### PR TITLE
Fix React errors caused by empty values in validation

### PR DESCRIPTION
**Pull request by ShaunYearStrong**

### CODE REVIEW

File: public/video-ui/src/constants/videoEditValidation.js [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
-  }
-  if (!values.category) {
-    errors.category = 'Required'
-  }
-  if (!values.posterImage) {
-    errors.posterImage = 'Required'
-  }
-  if (!values.duration) {
-    errors.duration = 'Required'
+  if (values) {
```
Comment (clloyd) [inline public/video-ui/src/constants/videoEditValidation.js]: Would this be nicer just to have a single `if (!values){return false}` at the top?

File: public/video-ui/src/actions/YoutubeActions/getCategories.js [hunk 2]
```diff
@@ -16,6 +16,7 @@ function receiveCategories(categories) {
 }
 
 function errorReceivingCategories(error) {
+  console.error(error);
```
Comment (akash1810) [inline public/video-ui/src/actions/YoutubeActions/getCategories.js]: is this needed?

Comment (clloyd) [inline public/video-ui/src/actions/YoutubeActions/getCategories.js]: Yeah

File: public/video-ui/src/constants/videoEditValidation.js (hunk 1 above)
Comment (ShaunYearStrong) [inline public/video-ui/src/constants/videoEditValidation.js]: Yeah actually, stops the nested `if`

**Pull request merged**